docs/sources/analyzing-profiles-in-pyroscope-ui/_index.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "How to use the Pyroscope UI for performance analysis"
+menuTitle: "How to use the Pyroscope UI"
+description: "How to use the Pyroscope UI to analyze performance of your applications."
```

Comment:
Instead of "How to" what about "Use Pyroscope for performance analysis"?  What does UI refer to? The Grafana > Pyroscope plugin UI or something else? Do we need to state UI specifically?

Also, I actually like the folder title. You could have a page that focuses on actually analyzing profiles and a second page that explains the UI. You'd have a task-oriented page and a reference page then. 

Headings should also avoid gerunds, so instead of "Analyzing profiles...", you'd use "Analyze profiles..."

```suggestion
title: Analyze app performance using Pyroscope  
menuTitle: Analyze app performance
description: How to use the Pyroscope UI to analyze performance of your applications.
```